airsonic-main/src/main/kotlin/org/airsonic/player/controller/api/SystemController.kt

```diff
@@ -0,0 +1,19 @@
+package org.airsonic.player.controller.api
+
+import org.airsonic.player.dto.PingResponse
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseBody
+
+@Controller
```

Comment:
Wouldn't it be preferable to use real @RestController as specified by Spring Rest ? (see https://spring.io/guides/gs/rest-service/)

Reply:
Yeah, that is a good idea.